src/main/java/com/powsybl/balances_adjustment/data_exchange/StandardProcessType.java

```diff
@@ -4,7 +4,7 @@
  * License, v. 2.0. If a copy of the MPL was not distributed with this
  * file, You can obtain one at http://mozilla.org/MPL/2.0/.
  */
-package com.powsybl.balances_adjustment.pevf;
+package com.powsybl.balances_adjustment.data_exchange;
 
 /**
  * Pan European Verification Function.
```

Comment:
Maybe replace the javadoc here by `Electronic Data Interchange`? (same remark for javadoc of files used for both imports)